OasisDEX/oasis-borrow
# Adjust Aave SL calculations

# [Adjust Aave SL calculations](https://app.shortcut.com/oazo-apps/story/7310/adjust-aave-sl-calculations-on-ui)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- adjusted aave sl calculations
- extracted LTV card and used both in aave overview and stop loss 
- removed unused, old details cards
  
## How to test 🧪
  <Please explain how to test your changes>

- create aave eth/usdc multiply vault using hardhat
- all numbers in stop loss details section and form should be correct (including slider min / max)
- double check whether SL numbers on maker are still fine
